static/scss/course-cards.scss

```diff
@@ -0,0 +1,106 @@
+.course-section-header {
```

Comment:
1. I don't think we need this class. You can add a class to the `h1` and apply any necessary spacing to that class
2. Please refer to my [comment on the other PR](https://github.com/mitodl/mitxonline/pull/45#discussion_r681984316) about header tags (`h1`, `h2`, etc). Please make absolutely sure that the custom styles are necessary. If the styling of this h1 in the designs is anywhere close to the standard h1/h2 styles that we have for attributes like size, weight, color, font, etc., please remove those styles here.
3. Please get rid of all `font-family: Montserrat` definitions, and make sure that all these `font-size` definitions are necessary. Just in general I think a lot of these styles might not need to be here